odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -597,13 +604,15 @@ PROCEDURE READCONFIGFILESETTINGS
 **  TC  PARAMCUSTOMTERMS      Custom terms and conditions
 **  FD  PARAMFEEDISCL         Custom fee disclosure
 **  RI  PARAMREVOKEINST       Revocation instructions
-**  OIO PARAMOPTINOUTINFO     Opt-in/Opt-out Information Text 
+**  OIO PARAMOPTINOUTINFO     Opt-in/Opt-out Information Text
 **  SI  PARAMSERVICEINST      Service Instuctions
```

Comment:
@tkainz Code looks good, Approved!
Only thing I found was spelling of Instuctions around line 608 "** SI PARAMSERVICEINST Service Instuctions"